tidb-cloud/migration-overview-for-cloud.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Migration Overview
+summary: Learn an overview of data migration scenarios and the solutions for TiDB Cloud.
+---
+
+# Migration Overview
+
+You can migrate data from a wide variety of data sources to TiDB Cloud. This document gives an overview of the data migration scenarios.
+
+## Configure Amazon S3 access and GCS access
+
+If your source data is stored in Amazon S3 or GCS buckets, before importing or migrating the data to TiDB Cloud, you need to configure cross-account access to the buckets. For more information, see [Configure Amazon S3 access and GCS access](/tidb-cloud/config-s3-and-gcs-access.md).
```

Comment:
GCS Bucket
configure cross-account access to the buckets  
It is recommended to change to   Google Cloud Storage (GCS) ，configure  access to the buckets

Reply:
OK. Updated.